### PR TITLE
Added hall of fame to README.md  #10

### PR DESCRIPTION
**Pull request by coderquill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,3 +46,17 @@ Transfer the apk to your device and install it to use the application.
 
 ## Note 
 * Marshmallow runtime permissions support not added. You need to enable permission manually in app settings for this app to work
+
+## Contributors
+[![](https://sourcerer.io/fame/arshadkazmi42/arshadkazmi42/android-contacts-scifiui/images/0)](https://sourcerer.io/fame/arshadkazmi42/arshadkazmi42/android-contacts-scifiui/links/0)
+[![](https://sourcerer.io/fame/arshadkazmi42/arshadkazmi42/android-contacts-scifiui/images/1)]
```
Comment (arshadkazmi42) [inline README.md]: Something looks broken here. I think the `line break` added here is breaking it.
Can you remove the the line breaks. Or copy paste the exact code as given in the issue and paste it here and update the PR

**Comment by arshadkazmi42** [comment]
@coderquill 
Thank you for taking this up and Congrats on your first PR.

I have added a comment for a small change, I think once that change is done, it will be ready to land in master

Also small advice, its good practice to create new feature branch if you are working on something, its not good practice to work on master branch.

File: README.md (hunk 1 above)
Comment (coderquill) [reply README.md]: Oh, okay. Will do it  .

**Comment by coderquill** [comment]
Thank you @arshadkazmi42. I will keep that in mind :)

**Comment by arshadkazmi42** [comment]
Looks good now. Merging it master. 
Congrats on landing your first patch 👍

**Comment by arshadkazmi42** [comment]
@coderquill Thanks alot for working on this

**Comment by coderquill** [comment]
Yay!! Thank you!

**Pull request merged**